packages/next/src/build/webpack/loaders/next-edge-ssr-loader/index.ts

```diff
@@ -255,12 +257,22 @@ const edgeSSRLoader: webpack.LoaderDefinitionFunction<EdgeSSRLoaderQuery> =
 
     export const ComponentMod = pageMod
 
-    export default function(opts) {
-      return adapter({
+    export default async function(opts, event) {
+      const res = await adapter({
```

Comment:
Why await? It doesn't hurt, but also isn't needed.

Reply:
We're using `res.waitUntil` down below, otherwise it will be a promise here.